### ISSUE TITLE
A1111 WEBUI became unusable in offline mod with sd-dynamic-prompts installed

### ISSUE THREAD

**Issue by Onkitova** [adieyal/sd-dynamic-prompts#175]
The thing is -- after recent update of sd-dynamic-prompts -- my WEBUI became unusable when there is no internet connection. 
I suspect that the problem is caused by forceful dependency installation which occurs on EACH start up of WEBUI now somewhy.

These 2 exactly:
![ldQNtCZX6T](https://user-images.githubusercontent.com/119720012/211437509-0a5682e9-4519-4e38-8454-c69241051e0e.png)

Also, there is a strange (log?) file reappearing on each run in a WEBUI parent folder called "0.2.5", without file extension.

Here it is:
[0.2.5.zip](https://github.com/adieyal/sd-dynamic-prompts/files/10378063/0.2.5.zip)

Please, fix this. I'd like to continue using your great WEBUI extension.

**Comment by adieyal** [comment]
The force update was there temporarily while there were many changes taking place on the dynamicprompts package and I didn't want to release a new version of the extension every time that happened. I think updates have slowed so I'll remove it. Doesn't make sense that it makes webui unstable though. I haven't heard anyone else mentioning it.

**Comment by Onkitova** [comment]
>Doesn't make sense that it makes webui unstable though. I haven't heard anyone else mentioning it.

I guess it could be easily reproduced. Just try to disable your internet connection somehow (firewall'ing should work too) and then run WEBUI. In my case, doing so is resulting in:
![KmiwlQTKiR](https://user-images.githubusercontent.com/119720012/211473027-e695f15f-c751-4664-b7c0-fa1676488509.png)
hanging for minute or two. Which is followed by one-shot huge errors log right in a command prompt:

[command prompt error log in webui.txt](https://github.com/adieyal/sd-dynamic-prompts/files/10379411/command.prompt.error.log.in.webui.txt)

It might look like some problem on a WEBUI side, but the fact I am facing is: whatever it actually is -- it is triggered by sd-dynamic-prompts installation. I tried to turn off (through moving folder) all other WEBUI extensions. The same for various WEBUI-specific command line args. Nothing helps, tho.
Then I also tried to turn off only sd-dynamic-prompts and... it fixed the launch problem. 

Yet, once again, when internet connection is on, it always work like that:
![image](https://user-images.githubusercontent.com/119720012/211475708-8c603cab-d878-46a1-96c8-a3b9bf9a5139.png)
after which WEBUI starts as expected, almost in no time.
